diff --git a/config.h b/config.h
index 18b287b..9361f4a 100644
--- a/config.h
+++ b/config.h
@@ -3,7 +3,7 @@ static const Block blocks[] = {
 	/*Icon*/	/*Command*/		/*Update Interval*/	/*Update Signal*/
 	/* {"⌨", "kbselect", 0, 30}, */
 	{"", "cat /tmp/recordingicon 2>/dev/null",	0,	9},
-	/* {"",	"music",	0,	11}, */
+	{"",	"music",	0,	11},
 	{"",	"pacpackages",	0,	8},
 	{"",	"news",		0,	6},
 	/* {"",	"crypto",	0,	13}, */
@@ -14,8 +14,8 @@ static const Block blocks[] = {
 	/* {"",	"price eth Ethereum 🍸",	9000,	23}, */
 	/* {"",	"price btc Bitcoin 💰",				9000,	21}, */
 	/* {"",	"torrent",	20,	7}, */
-	/* {"",	"memory",	10,	14}, */
-	/* {"",	"cpu",		10,	18}, */
+	{"",	"memory",	10,	14},
+	{"",	"cpu",		10,	18},
 	/* {"",	"moonphase",	18000,	17}, */
 	{"",	"forecast",	18000,	5},
 	{"",	"mailbox",	180,	12},
