--- rpios_dt.empty_string.dts	2024-12-27 11:15:03.786201748 +0100
+++ nixos.d4f51a4.dts	2024-12-27 11:05:25.648567878 +0100
@@ -489,43 +499,43 @@
 			};
 
 			emmc-gpio34 {
 				brcm,pull = <0x00 0x02 0x02 0x02 0x02 0x02>;
 				brcm,function = <0x07>;
 				phandle = <0x46>;
 				brcm,pins = <0x22 0x23 0x24 0x25 0x26 0x27>;
 			};
 
 			bt_pins {
-				brcm,pull;
-				brcm,function;
+				brcm,pull = <0x00>;
+				brcm,function = <0x04>;
 				phandle = <0x0a>;
-				brcm,pins;
+				brcm,pins = <0x2b>;
 			};
 
 			uart0-gpio36 {
 				brcm,function = <0x06>;
 				phandle = <0x5b>;
 				brcm,pins = <0x24 0x25>;
 			};
 
 			i2c1-gpio2 {
 				brcm,function = <0x04>;
 				phandle = <0x4e>;
 				brcm,pins = <0x02 0x03>;
 			};
 
 			uart0_pins {
-				brcm,pull;
-				brcm,function;
+				brcm,pull = <0x02 0x00 0x00 0x02>;
+				brcm,function = <0x07>;
 				phandle = <0x09>;
-				brcm,pins;
+				brcm,pins = <0x1e 0x1f 0x20 0x21>;
 			};
 
 			gpclk2-gpio6 {
 				brcm,function = <0x04>;
 				phandle = <0x4b>;
 				brcm,pins = <0x06>;
 			};
 
 			gpclk2-gpio43 {
 				brcm,pull = <0x00>;
@@ -606,21 +616,21 @@
 		power {
 			#power-domain-cells = <0x01>;
 			compatible = "raspberrypi,bcm2835-power";
 			firmware = <0x06>;
 			phandle = <0x10>;
 		};
 
 		hvs@7e400000 {
 			interrupts = <0x02 0x01>;
 			compatible = "brcm,bcm2835-hvs";
-			status = "okay";
+			status = "disabled";
 			reg = <0x7e400000 0x6000>;
 			phandle = <0x7e>;
 		};
 
 		csi@7e800000 {
 			power-domains = <0x10 0x0c>;
 			#address-cells = <0x01>;
 			clock-names = "lp", "vpu";
 			interrupts = <0x02 0x06>;
 			clocks = <0x08 0x2d 0x18 0x04>;
@@ -670,39 +680,39 @@
 			phandle = <0x7b>;
 		};
 
 		serial@7e215040 {
 			pinctrl-names = "default";
 			pinctrl-0 = <0x12>;
 			interrupts = <0x01 0x1d>;
 			clocks = <0x11 0x00>;
 			skip-init;
 			compatible = "brcm,bcm2835-aux-uart";
-			status = "disabled";
+			status = "okay";
 			reg = <0x7e215040 0x40>;
 			phandle = <0x26>;
 
 			bluetooth {
 				local-bd-address = [00 00 00 00 00 00];
 				fallback-bd-address;
 				shutdown-gpios = <0x07 0x2a 0x00>;
 				max-speed = <0x38400>;
 				compatible = "brcm,bcm43438-bt";
 				status = "disabled";
 				phandle = <0x3a>;
 			};
 		};
@@ -921,47 +930,47 @@
 		hdmi@7e902000 {
 			power-domains = <0x10 0x05>;
 			hpd-gpios = <0x07 0x1c 0x01>;
 			clock-names = "pixel", "hdmi";
 			reg-names = "hdmi", "hd";
 			interrupts = <0x02 0x08 0x02 0x09>;
 			clocks = <0x18 0x09 0x18 0x0d>;
 			ddc = <0x19>;
 			dma-names = "audio-rx";
 			compatible = "brcm,bcm2835-hdmi";
-			status = "okay";
+			status = "disabled";
 			reg = <0x7e902000 0x600 0x7e808000 0x100>;
 			phandle = <0x37>;
 			dmas = <0x0b 0x9000011>;
 		};
 
 		serial@7e201000 {
 			arm,primecell-periphid = <0x241011>;
 			pinctrl-names = "default";
-			pinctrl-0 = <0x09>;
+			pinctrl-0 = <0x09 0x0a>;
 			clock-names = "uartclk", "apb_pclk";
 			cts-event-workaround;
 			interrupts = <0x02 0x19>;
 			clocks = <0x08 0x13 0x08 0x14>;
 			skip-init;
 			compatible = "arm,pl011", "arm,primecell";
 			status = "okay";
 			reg = <0x7e201000 0x200>;
 			phandle = <0x25>;
 
 			bluetooth {
-				local-bd-address = [28 e6 85 eb 27 b8];
+				local-bd-address = [00 00 00 00 00 00];
 				fallback-bd-address;
 				shutdown-gpios = <0x07 0x2a 0x00>;
 				max-speed = <0x2dc6c0>;
 				compatible = "brcm,bcm43438-bt";
-				status = "disabled";
+				status = "okay";
 				phandle = <0x39>;
 			};
 		};
 
 		mmc@7e300000 {
 			pinctrl-names = "default";
 			pinctrl-0 = <0x13>;
 			bus-width = <0x04>;
 			interrupts = <0x02 0x1e>;
 			clocks = <0x08 0x1c>;
@@ -1170,75 +1174,50 @@
 		i2c1 = "/soc/i2c@7e804000";
 		spi0 = "/soc/spi@7e204000";
 		random = "/soc/rng@7e104000";
 		aux = "/soc/aux@7e215000";
 		gpio = "/soc/gpio@7e200000";
 		mmc1 = "/soc/mmcnr@7e300000";
 		dma = "/soc/dma-controller@7e007000";
 		mailbox = "/soc/mailbox@7e00b880";
 		soc = "/soc";
 		sdhost = "/soc/mmc@7e202000";
-		serial1 = "/soc/serial@7e215040";
+		serial1 = "/soc/serial@7e201000";
 		leds = "/leds";
 		uart0 = "/soc/serial@7e201000";
 		mmc = "/soc/mmc@7e300000";
 		fb = "/soc/fb";
 		i2c2 = "/soc/i2c@7e805000";
 		spi1 = "/soc/spi@7e215080";
 		i2s = "/soc/i2s@7e203000";
 		bluetooth = "/soc/serial@7e201000/bluetooth";
 		usb = "/soc/usb@7e980000";
 		i2c0 = "/soc/i2c0mux/i2c@0";
 		sound = "/soc/sound";
 		mmc0 = "/soc/mmc@7e202000";
 		phandle = <0x31>;
 		axiperf = "/soc/axiperf";
-		display0 = "/chosen/framebuffer@1eaa9000";
 		uart1 = "/soc/serial@7e215040";
-		serial0 = "/soc/serial@7e201000";
+		serial0 = "/soc/serial@7e215040";
 		watchdog = "/soc/watchdog@7e100000";
 		i2c = "/soc/i2c@7e804000";
 		i2c10 = "/soc/i2c0mux/i2c@1";
 	};
 
 	chosen {
